srcpkgs/SDRPlusPlus/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'SDRPlusPlus'
+pkgname=SDRPlusPlus
+version=1.0.4
+revision=1
+build_style=cmake
+configure_args="-DCMAKE_BUILD_TYPE=Release -DOPT_BUILD_AIRSPYHF_SOURCE=OFF
+ -DOPT_BUILD_PLUTOSDR_SOURCE=OFF"
+hostmakedepends="cmake pkg-config"
+makedepends="glfw-devel librtlsdr-devel glew-devel libhackrf-devel libusb-devel
+ SoapySDR-devel libairspy-devel rtaudio-devel alsa-lib-devel pulseaudio-devel
+ jack-devel fftw-devel volk-devel"
```

Comment:
it might not be optimal to have this depend on alsa, pulse, *and* jack (assuming those become shlib deps for the final package)

Reply:
I think it's ok, it will only depend on libjack, libpulseaudio and alsa-lib

Reply:
It does not seem to pick up either of them as runtime dependencies, so I guess the user will have to install the audio packages they want to use themselves. Is my observations correct?

`=> SDRPlusPlus-1.0.4_1: running pre-pkg hook: 999-collected-rdeps ...
   libstdc++>=4.4.0_1 libgcc>=4.4.0_1 glibc>=2.32_1 volk>=2.5.0_1 rtaudio>=5.1.0_1 SoapySDR>=0.7.0_1 libairspy>=1.0.9_1 librtlsdr>=0.5.3_1 libhackrf>=2017.02.1_1 libglvnd>=1.3.0_1 glew>=2.2.0_1 libfftw>=3.3_1 glfw>=3.0.4_1 
`

Reply:
then it probably `dlopen()`s the audio shlibs,and the user would have to install one themself.

Reply:
> then it probably `dlopen()`s the audio shlibs,and the user would have to install one themself.

That's perfect then :)